utils.js

```diff
@@ -1,11 +1,147 @@
 'use strict';
 
-var utils = {
+/**
+ * Uses abstract comparison (==) NOT strict comparison (===)
+ */
+function contains(list, item) {
```

Comment:
@odino, @shidhinrc and I spoke and I'm going to follow these guidelines:

- we will use lodash
- whoever uses this lib would include lodash otherwise things are gonna break :)